### PR TITLE
Remove redundant else keywords

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

**Comment by slide** [comment]
Can you resolve the conflicts?

**Comment by gpetrou** [comment]
I updated the PR a few days ago. Do you need something else?

**Comment by slide** [comment]
Sorry, just need to review it in detail since it touches a lot of stuff.

**Comment by gpetrou** [comment]
Oh OK, wasn't sure if you saw the update or not.

File: Src/Microsoft.Dynamic/Interpreter/LoopCompiler.cs [hunk 1]
```diff
@@ -262,33 +261,35 @@ private struct LoopVariable {
...
                 box = existing.BoxStorage;
                 _loopVariables[node] = new LoopVariable(existing.Access | access, box);
             } else if (_outerVariables.TryGetValue(node, out LocalVariable loc) || 
                 (_closureVariables != null && _closureVariables.TryGetValue(node, out loc))) {
+
                 // not tracking this variable yet, but defined in outer scope and seen for the 1st time
                 box = loc.InClosureOrBoxed ? Expression.Parameter(typeof(StrongBox<object>), node.Name) : null;
                 _loopVariables[node] = new LoopVariable(access, box);
             } else {
+
```
Comment (slide) [inline Src/Microsoft.Dynamic/Interpreter/LoopCompiler.cs]: Looks like some extra whitespace added her? Just a nitpick

**Pull request merged**